src/style.css

```diff
@@ -0,0 +1,64 @@
+@font-face {
+  font-family: Avenir-Book; /* Имя шрифта */
+  src: url(./fonts/macfonts/Avenir/Avenir-Book.ttf);
+}
+
+.search_bar_form {
+  margin-top: 20px;
+}
+
+.search_bar {
+  width: 100%;
+  font-weight: 300;
+  font-family: Avenir-Book, serif;
+  font-style: normal;
+  background-image: url("./images/Search.svg"), linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  background-repeat: no-repeat;
+  color: #3d4e61;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.big_input {
+  height: 70px;
+  background-position: top 25px left 26px;
+  background-size: 19px 19px;
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.small_input {
+  height: 42px;
+  position: relative;
+  background-position: top 12px left 14px;
+  background-size: 11px 11px;
+  padding-left: 33px;
+  font-size: 14px;
+  line-height: 19px;
+}
```

Comment:
We don't have any basic search bar, so why we should use there big and small search class names? We have search-input and search-input - small (--small). Therefore, we set all styles that are supposed to be for the "large" input for all inputs in general, and with the modifier small we change only the necessary values to make our input smaller.